modules/src/ics02_client/handler/create_client.rs

```diff
@@ -62,24 +63,29 @@ pub fn keep(keeper: &mut dyn ClientKeeper, result: CreateClientResult) -> Result
 mod tests {
     use super::*;
     use crate::ics02_client::context_mock::MockClientContext;
+    use crate::ics03_connection::msgs::test_util::get_dummy_account_id;
+    use crate::ics07_tendermint::client_state::ClientState;
     use crate::ics07_tendermint::header::test_util::get_dummy_header;
-    use crate::ics07_tendermint::msgs::create_client::MsgCreateClient;
     use crate::mock_client::header::MockHeader;
     use crate::mock_client::state::{MockClientState, MockConsensusState};
-    use std::str::FromStr;
+    use std::str::{from_utf8, FromStr};
```

Comment:
I would rather have `use std::str::{self, FromStr}` and then use `str::from_utf` at call site to make it clearer what we are converting the utf8-encoded bytes into.

```suggestion
    use std::str::{self, FromStr};
```